contrib/mesos/pkg/executor/executor.go

```diff
@@ -165,12 +150,14 @@ func New(config Config) *KubernetesExecutor {
 		shutdownAlert:        config.ShutdownAlert,
 		exitFunc:             config.ExitFunc,
 		podStatusFunc:        config.PodStatusFunc,
-		initialRegComplete:   make(chan struct{}),
 		staticPodsConfigPath: config.StaticPodsConfigPath,
 		launchGracePeriod:    config.LaunchGracePeriod,
 	}
 
 	// watch pods from the given pod ListWatch
+	if config.PodLW == nil {
+		log.Fatal("cannot create executor with nil PodLW")
```

Comment:
`panic()` here instead of `Fatal()` so that we can at least unit test this at some point